examples/invoke.rs

```diff
@@ -61,7 +80,8 @@ fn main() {
 
         interpreter::ExecutionParams::from(args)
     };
-    
+
+    // Intialize deserialized module.
```

Comment:
This could be much more detailed description of what's going on (why `"main"`, why `None`), what going one when invoking `add_module` (validation and so on)